File: Logic Building Problems/Basic Problems/Program to print multiplication table of a number/logic1.java
// Java program to print table of a number with iterative approach
import java.io.*;

class logic1{

    public static void printTable(int n)  {
              
        for (int i = 1; i <= 10; ++i) 
            System.out.println(n + " * " + i +
                               " = " + n * i);
    }
  
    public static void main(String arg[]){   
        int n = 5; 
        printTable(n);
    }
}

File: Logic Building Problems/Basic Problems/Program to print multiplication table of a number/logic1.cpp
// CPP program to print table of a number
#include <iostream>
using namespace std;

void printTable(int n) { 
  for (int i = 1; i <= 10; ++i) 
        cout << n << " * " << i << " = "  
             << n * i << endl;
}

int main() {
    int n = 5;  
    printTable(n);
    return 0;
}